Source/Scene/Cesium3DTileOptimizations.js

```diff
@@ -0,0 +1,125 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/defaultValue',
+        '../Core/freezeObject',
+        './TileOrientedBoundingBox',
+        './TileBoundingRegion'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        freezeObject,
+        TileOrientedBoundingBox,
+        TileBoundingRegion) {
+    'use strict';
+
+    /**
+     * @alias Cesium3DTileOptimizationFlag
+     *
+     * Flag denoting support for a 3D Tiles optimization
+     */
+    var Cesium3DTileOptimizationFlag = {
+        UNSUPPORTED: 0,
+        SUPPORTED: 1,
+        INDETERMINATE: -1
+    };
+
+    /**
+     * Defines optional optimization flags for a {@link Cesium3DTileset}.
+     * Optimizations marked as Cesium3DTileOptimizations.Flags.SUPPORTED or Cesium3DTileOptimizations.Flags.UNSUPPORTED
```

Comment:
Use `{@link }` when referring to enums.

For more, see the [doc guide](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/DocumentationGuide/README.md#documentation-guide).